roles/kubernetes-apps/ansible/tasks/main.yaml

```diff
@@ -5,6 +5,7 @@
   until: result.status == 200
   retries: 10
   delay: 6
+  when: inventory_hostname == groups['kube-master'][0]
```

Comment:
That is true if you run it from cluster.yml, but if you run it as role.  It doesn't necessarily.